.travis.yml

```diff
@@ -26,28 +26,32 @@ script:
 - docker tag todo-app:prod legendarypan/todo-app:prod
 - docker push legendarypan/todo-app:prod
 
-# Pre-deploy Install
-- curl -sS https://cli-assets.heroku.com/install.sh | sh
+# Pre-deploy Install - removed post module 11
+# - curl -sS https://cli-assets.heroku.com/install.sh | sh
 # - heroku update
 # - heroku plugins:install @heroku-cli/plugin-container-registry
 
 
-# using heroku credentials
+# using heroku credentials - commented out post module 11, migrated app from Heroku to Azure cloud
 # NOTES use auth token instead of API key and move to environment variable like docker_login 
-- docker login --username=$HEROKU_LOGIN --password=$HEROKU_API_KEY registry.heroku.com
-- docker tag legendarypan/todo-app:prod registry.heroku.com/fengs-todo-app/web
-- docker push registry.heroku.com/fengs-todo-app/web
+# - docker login --username=$HEROKU_LOGIN --password=$HEROKU_API_KEY registry.heroku.com
+# - docker tag legendarypan/todo-app:prod registry.heroku.com/fengs-todo-app/web
+# - docker push registry.heroku.com/fengs-todo-app/web
 
-# Trying diff deploy method
 deploy:
   provider: script
-  script: heroku container:release web --app fengs-todo-app
-  # heroku container:push web --app fengs-todo-app
+  # removed below heroku post module 11
+  # script: heroku container:release web --app fengs-todo-app
+  script: 
+    # curl -dH -X POST "https://\$fengwebapp:SaELzMgX5YlJkpPfHHh1MlsS1QWL3Q2eAibgldjtE9KLLBdRtYsSl961EdaL@fengwebapp.scm.azurewebsites.net/docker/hook"
+    # curl -dH -X POST "$AZURE_WEBHOOK"
+    bash webhook.sh
   on:
   #  all_branches: true
   #  branch: module_8
   #  branch: master # (revert back to this after merging module_9)
-     branch: module_10
+     branch: module_11
```

Comment:
Let's clean out the old comments, and don't forget to revert back to master when merging